dex/testing/dcr/harness.sh

```diff
@@ -215,26 +218,30 @@ sleep 3
 ################################################################################
 
 echo "Creating simnet alpha wallet"
+USE_SPV="0"
```

Comment:
On the alpha wallet or beta? Can spv wallets vote on tickets? Can't remember.

Reply:
Hmm, good point about ticket voting.  I don't believe SPV can vote on tickets.  I tend to use beta when testing anyway.  Let me get confirmation about the voting thing.

Reply:
Alright, that SPV an option only for beta then.  It doesn't seem likely that voting will be allowed on SPV wallets because a sensible block validation process isn't gonna be possible short of pulling the blocks.